src/App/pages/MainPage/components/Search/Search.tsx

```diff
@@ -12,27 +14,39 @@ interface Props {
 
 const Search: FC<Props> = ({ className }) => {
   const [value, setValue] = useState<Option[]>([]);
-  const [querry, setQuerry] = useState<string>('');
+  const [text, setText] = useState<string>('');
```

Comment:
[+] Можно не усложнять излишне код, если и зрительно, и для typescript'а ясно, какой здесь будет тип:
```suggestion
  const [text, setText] = useState('');
```